draft-documents/satisfying-real-world-use-cases.md

```diff
@@ -247,14 +247,13 @@ Everywhere Else, Peruvian economist Hernando de Soto highlights a fundamental
 difference between Western liberal democracies and other regions: such
 democracies have functioning systems that enable the abstract representation of
 value. For example, land is not simply owned territory--it is private property
-registered with the state and represented by land deeds; the representation
-having the force of law in disputes.
+registered with the state and represented by land deeds; This abstrct representation of the value of the land that is owned, the deed, is the instramental tool used to access capital. The west also has functioning state institutions and systems of law that imbue the deeds with cultural and legal power. They have what could be considered high levels of social trust - as described by Steven M.R. Covey in The Speed of Trust and summarized by one of the co-authors oof this document - Kaliya Identity Woman http://www.identitywoman.net/the-trouble-with-trust-the-case-for-accountability-frameworks
 
-Millions of individuals do not have such representative records (birth
+Millions of individuals do not have such representative records of themselves (birth
 certificates etc.) registered with government entities, and such entities may
 not be reliable enough to serve this role. Worse, many marginalized people may
 either be prohibited from creating records that might enfranchise them, either
-through legal exclusion or threats of violence. The Gates Foundation considers
+through legal exclusion or threats of violence. Without these types of records their access to state and commercial services (such as banking) is limited. The Gates Foundation considers
```

Comment:
It is interesting... I don't know that the regular reader necessarily knows what "enfranchisement" in this case actually means - or translates to in practical way.